compiler-rt/lib/rtsan/rtsan.cpp

```diff
@@ -49,7 +50,21 @@ static auto OnViolationAction(DiagnosticsInfo info) {
   return [info]() {
     IncrementTotalErrorCount();
 
-    PrintDiagnostics(info);
+    BufferedStackTrace stack;
+    stack.Unwind(info.pc, info.bp, nullptr,
+                 /*request_fast*/ true);
+
+    StackDepotHandle handle = StackDepotPut_WithHandle(stack);
```

Comment:
Theoretically, if we run another sanitizer and RTSan at the same time (can you do that?) this will be wrong. Do we need a separate instance of StackDepot for this? Or do we need to keep count separately?

Reply:
Currently rtsan is incompatible with all other sanitizers. We will have to keep this in mind for any future interop between us and another runtime.

This inserts into the "global/static stack depot" but some of the sanitizers that place nice with one another have their own (scudo_stackdepot.h, msan_chained_origin/dfsan_chained_origin)

Reply:
if there is any chance for that interop in the future, please leave a comment in the code about this.